docs/distributions/debian/installation.md

```diff
@@ -1,93 +1,97 @@
 * External keyboard and mouse
 
 # Install Procedure
 
-!!! Warning "Installation fails when installing the bootloader (Eg :- GRUB)"
-    It is possible that during installation, when the installer is installing the bootloader (GRUB in most cases), the installation may fail. This is because the distro's ISO is using an old Linux kernel which doesn't support writing to the NVRAM of T2 Macs. In such a case, boot into the ISO again. When the initial menu gets displayed having options to try/install the distro, press "e" on the option you otherwise would have chosen to install. This will open the command line. Add `efi=noruntime` to the command line and press "F10" to boot. This should fix the issue.
+!!! Warning "Installation fails when installing the bootloader (e.g. GRUB)"
+    It is possible that during installation, when the installer is installing the bootloader (GRUB in most cases), that the installation may fail. This is because the distro's ISO is using an old Linux kernel which doesn't support writing to the NVRAM of T2 Macs. In such a case, boot into the ISO again. When the initial menu gets displayed having options to try/install the distro, press "e" on the option you otherwise would have chosen to install. This will open the command line. Add `efi=noruntime` to the command line and press "F10" to boot. This should fix the issue.
 
 !!! Warning "Pop!_OS"
-    Due to a bug in Pop!_OS installer, the partition sizes shown by it are incorrect during manual partitioning step. As a workaround you may follow the instructions given in this [GitHub issue](https://github.com/elementary/installer/issues/620#issuecomment-1356978490) in the live ISO environment to fix the installer and then start the installation.
+    Due to a bug in the Pop!_OS installer, the partition sizes shown by it are incorrect when using manual partitioning. As a workaround you may follow the instructions given in this [GitHub issue](https://github.com/elementary/installer/issues/620#issuecomment-1356978490) in the live ISO environment to fix the installer and then start the installation.
 
 1. Follow the [Pre-installation](https://wiki.t2linux.org/guides/preinstall) steps to prepare your Mac for installation.
-2. Boot into the Live ISO. You should now be in the GRUB boot menu. Select the option which is relevent to you.
-3. Start the installer and install it like normal until you get an option to manually specify partitions.
-4. Find the partition you made for Linux when you were following the Pre-installation steps. MAKE SURE TO SELECT THE RIGHT PARTITION OR ELSE YOUR DATA WILL BE LOST. Delete it to make free space. You'll need to make these partitions:
+2. Boot into the Live ISO. In the GRUB boot menu, select the option relevant to you.
+3. Start the installer and go through it like normal until you get to partitioning. Select the relevant option for **manual partitioning**.
```

Comment:
Until you get to Partitioning looks as if the installer shall always provide you a screen with Partitioning. Normally installers push users to use automated installation imo.

Reply:
Edited!